src/components/BlogPage/Blogs.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+
+import { graphql, Link, useStaticQuery  } from "gatsby"
+
+const Blogs = () => {
+
+  const query = useStaticQuery(graphql`
+    {
+      allWpPost(sort: {order: DESC, fields: date}, skip: 1) {
+        edges {
+          node {
+            id
+            title
+            date(formatString: "YYYY-MM-DD")
+            slug
+          }
+        }
+      }
+    }
+  `)
+
+  const { edges } = query.allWpPost
+
+  console.log("edges", edges);
+
+  return (
+    <div>
+      <h2>The Latest</h2>
+      {
+        edges.map((edge) => {
```

Comment:
we can add `edges.length > 0` before the return.

Reply:
I will check that out. Thank you.

Reply:
Do you think this will work?
```js
const { edges } = query.allWpPost

  if (edges.length === 0) return <div></div>

  return (
    <div>
      <h2>The Latest</h2>
      {
```

Else, I might need to call another query on parent pages to check for the number of blogs and render accordingly. This way might be better.